.github/workflows/pnpm-wireit-release.yml

```
@@ -4,8 +4,54 @@ defaults:
   run:
     working-directory: ./pnpm-wireit
 
-# TODO(5): Add actions
+on:
+  push:
+    branches:
+      - main
+
 jobs:
-  build:
+  release:
+    runs-on: ubuntu-latest
+
     steps:
-      - run: "echo TODO IMPLEMENT CHANGESETS ACTION"
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18.x
```

NBD, but this should probably be 

```suggestion
          node-version: ${{ matrix.node-version }}
```